book/website/collaboration/ambassador-schemes.md

```diff
@@ -0,0 +1,34 @@
+(cl-ambassador-schemes)=
+# Ambassador Schemes
+
+(cl-ambassador-schemes-prerequisites)=
+# Prerequisites
```

Comment:
```suggestion
## Prerequisites
```

Reply:
The reason for the missing subchapters was that this heading was on the wrong level.